zerver/views/upload.py

```diff
@@ -156,6 +165,34 @@ def serve_local_file_unauthed(request: HttpRequest, token: str, filename: str) -
     return serve_local(request, path_id, url_only=False)
 
 
+def serve_local_avatar_unauthed(request: HttpRequest, path: str) -> HttpResponseBase:
+    if settings.LOCAL_UPLOADS_DIR is None:
```

Comment:
Should this be `LOCAL_AVATARS_DIR`? That might let us remove the assert just after.

Reply:
I think I was trying to always case on `LOCAL_UPLOADS_DIR` because `LOCAL_AVATARS_DIR` is a _derived_ value, but I think in retrospect that's not helpful.  Adjusted, and fixed a test mock to now also stub out `LOCAL_AVATARS_DIR` as needed.